freqtrade/rpc/api_server.py

```diff
@@ -83,6 +89,11 @@ def __init__(self, freqtrade) -> None:
 
         self._config = freqtrade.config
         self.app = Flask(__name__)
+
+        # Setup the Flask-JWT-Extended extension
+        self.app.config['JWT_SECRET_KEY'] = 'super-secret'  # Change this!
```

Comment:
Do you want to release it as it is?

Reply:
i'm not sure what difference this key vs. a different key makes.

It makes a different if we assume it's a website we deploy and want noone to login.
In this case however, we're giving this to our users, who then run this as server ... so no matter WHAT we set this to - it'll be the same in all cases.

Reply:
i'll move it to the configuration - so it's easily changable ... :)

Reply:
👍 
otherwise the hashes/cookies etc can be predictable, I assume...
no need to make an utility for changing/generating it, or other heavy stuff (yet)...